### DISCUSSION TITLE
I Change the VC Format braces settings in VSCode but not working

### DISCUSSION THREAD

**Discussion by FXGHH** [microsoft/vscode-cpptools#8585] [Q&A]
![屏幕截图 2021-12-20 015815](https://user-images.githubusercontent.com/89591028/146679622-366c46db-bb14-45f4-ac09-0f955c64fe57.jpg)
![屏幕截图 2021-12-20 015839](https://user-images.githubusercontent.com/89591028/146679624-d20ec0cb-0330-4916-8f2a-76d7e11d9a53.jpg)

**Comment by sean-mcmanus** [comment]
This appears to be https://github.com/microsoft/vscode-cpptools/issues/5538
